modules/gapi/src/backends/cpu/gcpuvideo.cpp

```diff
@@ -107,13 +107,59 @@ GAPI_OCV_KERNEL_ST(GCPUBackgroundSubtractor,
     }
 };
 
+GAPI_OCV_KERNEL_ST(GCPUKalmanFilter, cv::gapi::video::GKalmanFilter, cv::KalmanFilter)
+{
+    static void setup(const cv::GMatDesc&, const cv::GOpaqueDesc&,
+                      const cv::GMatDesc&, const cv::gapi::video::KalmanParams& kfParams,
+                      std::shared_ptr<cv::KalmanFilter> &state, const cv::GCompileArgs&)
+    {
+        GAPI_Assert(kfParams.type == CV_32F || kfParams.type == CV_64F);
```

Comment:
Do we have to check it one more time here? There is the same assert in `outMeta`

Reply:
Removed